src/Microsoft.Framework.Runtime/NuGet/Utility/VersionUtility.cs

```diff
@@ -80,6 +88,9 @@ public static class VersionUtility
         // These aliases allow us to accept 'wp', 'wp70', 'wp71', 'windows', 'windows8' as valid target farmework folders.
         private static readonly Dictionary<FrameworkName, FrameworkName> _frameworkNameAlias = new Dictionary<FrameworkName, FrameworkName>(FrameworkNameEqualityComparer.Default)
         {
+            { new FrameworkName("Asp.Net, Version=v5.0"), new FrameworkName("DNX, Version=v4.5.1") },
```

Comment:
Sure, but what parts of the system get affected? What happens when you kpm build and have both frameworks? What happens if you kpm build and have aspnetcore50? Does it translate into dnxcore50?